docs/en/tutorials/getting-started.rst

```diff
@@ -103,7 +103,7 @@ Install Doctrine using the Composer Dependency Management tool, by calling:
     $ composer install
 
 This will install the packages Doctrine Common, Doctrine DBAL, Doctrine ORM,
-Symfony YAML and Symfony Console into the `vendor` directory. The Symfony 
+Symfony YAML and Symfony Console into the `vendor` directory. The Symfony
```

Comment:
Maybe we should remove `Symfony YAML` from this page? I mean v3 already removed support for it, v2.7 deprecates it, and I think we should start phasing it out ASAP.

Can you please also update the version in L90?

Reply:
Yes, I will, I'm just waiting for a moment when I'll have more time for this PR, it's hard work 😉